pep-0008.txt

```diff
@@ -333,7 +333,7 @@ technical terms are used which aren't English). In addition, string
 literals and comments must also be in ASCII. The only exceptions are
 (a) test cases testing the non-ASCII features, and
 (b) names of authors. Authors whose names are not based on the
-latin alphabet MUST provide a latin transliteration of their
+Latin alphabet MUST provide a Latin transliteration of their
```

Comment:
I think this is really talking about the `latin-1` or `iso-8859-1` [encoding](https://docs.python.org/3/library/codecs.html#standard-encodings) so maybe those terms should be used instead.

Reply:
Fair enough. I'll change it, do you have a preference? 
For clarity/readability I'd probably preference `latin-1` or `iso-8859-1 (latin-1)`  over just the `iso-8859` reference.

Reply:
Personally, I prefer to talk about the alphabet, not the encoding. So "Latin" would be correct (capitalized IMO), but "latin-1" would not be.

Reply:
Possibly so, but concretely, it's non-latin-1 characters that do (or did) cause technical problems, so it's better to specific and discuss the encoding.

Reply:
How about `Latin alphabet (specifically the latin-1/ISO-8859-1 encoding)` - or something similar? Best of both worlds? Also - `ISO or `iso`?